hybrid/utils.py

```diff
@@ -64,10 +64,10 @@ def bqm_reduced_to(bqm, variables, sample, keep_offset=True):
         This example reduces a 3-variable BQM to two variables.
 
         >>> import dimod           # Create a binary quadratic model
-        >>> bqm = dimod.BinaryQuadraticModel({}, {('a', 'b'): -1, ('b', 'c'): -1, ('c', 'a'): -1}, 0, 'BINARY')
+        >>> bqm = dimod.BinaryQuadraticModel({}, {('a', 'b'): -1.0, ('b', 'c'): -1.0, ('c', 'a'): -1.0}, 0.0, 'BINARY')
         >>> sample = {'a': 1, 'b': 1, 'c': 0}
         >>> bqm_reduced_to(bqm, ['a', 'b'], sample)
-        BinaryQuadraticModel({'a': 0.0, 'b': 0.0}, {('a', 'b'): -1}, 0.0, Vartype.BINARY)
+        BinaryQuadraticModel({'a': 0.0, 'b': 0.0}, {('a', 'b'): -1.0}, 0.0, Vartype.BINARY)
```

Comment:
Are you sure variables in the first dictionary are going to be printed in the sorted order?

Reply:
That's a good point, it seems to me they always are but it is a dict

Reply:
BQMs are unordered, other than to the extend all dics are ordered in py3.6+ (RTD currently uses 3.7)

Reply:
Exactly. So either we use OrderedBQMs implicitly (not transparent), or we change all doctests (nor nice nor practical), or we accept tests will fail for users running them locally on older Pythons.

I think we can live with that last option.